app/src/forms/form/exportService.js

```diff
@@ -62,44 +62,64 @@ const service = {
     return `${form.snake()}_${type}.${format}`.toLowerCase();
   },
 
-  _submissionsColumns: (params = {}) => {
+  _submissionsColumns: (form, params = {}) => {
     if (params.columns) {
       return params.columns;
     }
-    return ['confirmationId', 'formName', 'version', 'createdAt', 'fullName', 'username', 'email', 'submission'];
+    // Custom columns not defined - return default column selection behavior
+    let columns = [
+      'confirmationId',
+      'formName',
+      'version',
+      'createdAt',
+      'fullName',
+      'username',
+      'email'
+    ];
+    // if form has 'status updates' enabled in the form settings include these in export
+    if (form.enableStatusUpdates) {
+      columns = columns.concat(['status', 'assignee', 'assigneeEmail']);
+    }
+    // and join the submission data
+    return columns.concat(['submission']);
   },
 
-  _getForm: async (formId) => {
+  _getForm: (formId) => {
     return Form.query().findById(formId).throwIfNotFound();
   },
 
-  _getData: async (exportType, formId, params = {}, currentUser) => {
+  _getData: async (exportType, form, params = {}, currentUser) => {
     if (EXPORT_TYPES.submissions === exportType) {
-      return await service._getSubmissions(formId, params, currentUser);
+      return await service._getSubmissions(form, params, currentUser);
     }
     return {};
   },
 
   _formatData: async (exportFormat, exportType, form, data = {}) => {
-    if (EXPORT_FORMATS.csv === exportFormat) {
-      if (EXPORT_TYPES.submissions === exportType) {
-        return await service._formatSubmissionsCsv(form, data);
+
+    // make headers/data fields more human readable
```

Comment:
Extra whitespace line on Line 99 should be removed.
Comment doesn't actually line up with following logic as it's always been human readable. Could consider saying that it's inverting content structure nesting to prioritize submission content clarity instead.